api-reference/50 Common/Object Structures/positionConfig/my/my.md

```diff
@@ -5,26 +5,113 @@ type: String | Object
 ---
 ---
 ##### shortDescription
-The position of the widget to align against the target element.
+Specifies a side or corner of the overlay element that should be aligned with the [target element](/Documentation/ApiReference/Common/Object_Structures/positionConfig/#of).
 
 ---
-The **my** option can take on an object containing the **x** and **y** fields, which specify horizontal and vertical position specifier respectively, or a string value consisting of horizontal and vertical position specifiers separated by a space (e.g., "left top"). The default value for each position specifier is "center". If you assign the "left" value to this option, it will be converted to the "left center" value.
 
-    <!--JavaScript-->
-    position: { my: 'left' };
+To set this option, use an object with the **x** and **y** fields or one of the string shortcuts listed in the accepted values.
```

Comment:
Can we be more specific than "use an object with the **x** and **y** fields"
(See my similar comments above)

"or one of the string shortcuts listed in the accepted values"
Change to "or a string shortcut from the accepted values list"